org.hedgewars.Hedgewars.yaml

```diff
@@ -0,0 +1,399 @@
+sdk: org.kde.Sdk
+command: hedgewars
+rename-desktop-file: hedgewars.desktop
+rename-icon: hedgewars
+rename-appdata-file: hedgewars.appdata.xml
+finish-args:
+- --socket=x11
+- --share=ipc
+- --socket=wayland
+- --socket=pulseaudio
+- --device=all
+- --share=network
+- --persist=.hedgewars
+modules:
+- shared-modules/glew/glew.json
+- shared-modules/glu/glu-9.0.0.json
+- name: lua51
+  sources:
+  - type: archive
+    url: https://www.lua.org/ftp/lua-5.1.5.tar.gz
+    sha256: 2640fc56a795f29d28ef15e13c34a47e223960b0240e8cb0a82d9b0738695333
+  - type: patch
+    path: lua51-cflags.patch
+  - type: patch
+    path: lua51-prefix.patch
+  - type: patch
+    path: lua51-so.patch
+  - type: shell
+    commands:
+    - sed -e 's:llua:llua5.1:' -e 's:/include:/include/lua5.1:' -i etc/lua.pc
+    - sed -r -e '/^LUA_(SO|A|T)=/ s/lua/lua5.1/' -e '/^LUAC_T=/ s/luac/luac5.1/' -i src/Makefile
+    - mv doc/lua.1 doc/lua5.1.1
+    - mv doc/luac.1 doc/luac5.1.1
+  cleanup:
+  - /bin
+  - /include
+  - /lib/*.a
+  - /lib/*.la
+  - /lib/pkgconfig
+  - /share/man
+  buildsystem: simple
+  build-commands:
+  - make CFLAGS='-O2 -Wall -fPIC' linux
+  - make TO_BIN='lua5.1 luac5.1' TO_LIB='liblua5.1.a liblua5.1.so liblua5.1.so.5.1
+    liblua5.1.so.5.1.5' INSTALL_TOP='/app' INSTALL_INC='/app/include/lua5.1' TO_MAN='lua5.1.1
+    luac5.1.1' INSTALL_MAN='/app/share/man/man1' install
+  - install -Dm0644 etc/lua.pc /app/lib/pkgconfig/lua51.pc
+  - ln -s liblua5.1.so /app/lib/liblua.so.5.1
+  - ln -s liblua5.1.so /app/lib/liblua.so.5.1.5
+- name: physicsfs
+  buildsystem: cmake-ninja
+  sources:
+  - type: archive
+    url: https://www.icculus.org/physfs/downloads/physfs-3.0.2.tar.bz2
+    sha256: 304df76206d633df5360e738b138c94e82ccf086e50ba84f456d3f8432f9f863
+  cleanup:
+  - /bin
+  - /include
+  - /lib/*.a
+  - /lib/pkgconfig
+- name: haskell
+  buildsystem: simple
+  sources:
+  - type: archive
+    url: https://downloads.haskell.org/~ghc/8.6.4/ghc-8.6.4-x86_64-deb9-linux.tar.xz
+    sha256: ef74222ef3c01c3fc5b926f67e8b4ef612fe8efa40ac937317cff9b0eed8d863
+    only-arches:
+    - x86_64
+  - type: archive
+    url: https://downloads.haskell.org/~ghc/8.6.4/ghc-8.6.4-i386-deb9-linux.tar.xz
+    sha256: 5e2ce88f4d13d23ac37e278e0c7b51c801008931359b9fa8a631d804d2da552c
+    only-arches:
+    - i386
+  build-commands:
+  - install -d /app/lib
+  - ln -s /usr/lib/*-linux-gnu/libtinfo.so.6 /app/lib/libtinfo.so.5
+  - ./configure --prefix=/app
+  -  make install
+  cleanup:
+  - '*'
+- name: haskell-primitive
+  buildsystem: simple
+  sources:
+  - type: archive
+    url: https://hackage.haskell.org/package/primitive-0.6.4.0/primitive-0.6.4.0.tar.gz
+    sha256: 4cbeaf7924dd79221f327ea101a29bf35c4976dc3319df157ff46ea68e6a0c64
+  - type: shell
+    commands:
+    - sed -e 's/< *4.12/<5/' -i primitive.cabal
+  build-commands:
+  - runhaskell Setup configure --prefix=/app
+  - runhaskell Setup build
+  - runhaskell Setup install
+  cleanup:
+  - '*'
+- name: haskell-vector
```

Comment:
Have you generated these dependencies somehow? It would be nice to have such tool in [flatpak-builder-tools](https://github.com/flatpak/flatpak-builder-tools/).

Reply:
No